app/src/main/java/org/simple/clinic/di/DateFormatter.kt

```diff
@@ -6,6 +6,9 @@ import javax.inject.Qualifier
 annotation class DateFormatter(val value: Type) {
```

Comment:
There's been a spelling mistake in the commit message. Should have been `FullYear` and not `FullyYear`

Reply:
Good catch. Resolved it 👍🏾